enterprise/utils.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""
+Utilities to be used across the enterprise djangoapp
+"""
+try:
+    from openedx.core.djangoapps.api_admin.utils import course_discovery_api_client
```

Comment:
There may be cases (for example, during testing) where we won't be in an EdX environment, but that's OK. Raising the exception when we go to create the API client, rather than when the import fails, allows us to load the package properly, and then mock out the appropriate calls for testing.